public/index.js

```diff
@@ -39,6 +51,18 @@ const onsubmit = function(ev) {
       .catch((err) => $markdown.value = 'Error: ' + err.toString());
 };
 
+const parseUrl = function() {
+  const query = location.search.replace(/^\?/, '').split('&');
+  const params = {};
+  query.forEach(function(tuple) {
+    const keyAndVal = tuple.split('=');
+    params[keyAndVal[0]] = keyAndVal[1];
+  });
+  console.log(params);
+  return params;
+};
+
+window.addEventListener('load', onPageLoad);
```

Comment:
Its certainly not bad, but shouldn't be needed since the script tag is the last thing parsed on the page, but i guess if we ever add async script loading it'll be good.